### PR TITLE
Added custom style for the "Delete" and "Save" buttons from the delete confirmation page.

### PR DESCRIPTION
**Pull request by natmil**
cofirmatio-delete

### CODE REVIEW

**Comment by natmil** [comment]
Well, not "Save" button, "Cancel" instead. 😁

**Comment by fabiocaccamo** [comment]
Could you explain what problem is fixed by this PR please?

**Comment by natmil** [comment]
> Could you explain what problem is fixed by this PR please?

Of course! The delete confirmation page didn't change the colors that you had configurated in the first place.

deletesave1

The cancel button was fine, but not the hover.

deletesave2

deletesave3

**Comment by fabiocaccamo** [comment]
@natmil perfect, thank you!

File: admin_interface/templates/admin_interface/css/admin-interface.css [hunk 1]
```diff
@@ -274,19 +274,22 @@
 .admin-interface .button.default:hover,
 .admin-interface input[type=submit].default:active,
 .admin-interface input[type=submit].default:focus,
-.admin-interface input[type=submit].default:hover {
+.admin-interface input[type=submit].default:hover,
+.delete-confirmation form .cancel-link:hover {
```
Comment (fabiocaccamo) [inline admin_interface/templates/admin_interface/css/admin-interface.css:278]: Please add always `.admin-interface` body class prefix to all selectors.

File: admin_interface/templates/admin_interface/css/admin-interface.css [hunk 2]
```diff
@@ -274,19 +274,22 @@
...
     background:{{ theme.css_save_button_background_hover_color }};
     color:{{ theme.css_save_button_text_color }};
     outline: none;
 }
 
 .admin-interface .submit-row a.deletelink:link,
-.admin-interface .submit-row a.deletelink:visited {
+.admin-interface .submit-row a.deletelink:visited,
+.delete-confirmation form input[type="submit"] {
```
Comment (fabiocaccamo) [inline admin_interface/templates/admin_interface/css/admin-interface.css:286]: Same as above.

File: admin_interface/templates/admin_interface/css/admin-interface.css [hunk 3]
```diff
@@ -274,19 +274,22 @@
...
     background:{{ theme.css_delete_button_background_color }};
     color:{{ theme.css_delete_button_text_color }};
 }
 
-.admin-interface .submit-row a.deletelink:hover {
+.admin-interface .submit-row a.deletelink:hover,
+.delete-confirmation form input[type="submit"]:hover {
```
Comment (fabiocaccamo) [inline admin_interface/templates/admin_interface/css/admin-interface.css:292]: Same as above.

**Review by fabiocaccamo — CHANGES_REQUESTED** [review]

**Comment by fabiocaccamo** [comment]
@natmil could you fix the selector prefix, please?

**Comment by merwok** [comment]
So this was merged without correcting the selector?

**Comment by fabiocaccamo** [comment]
@merwok yes, I fixed it [here](https://github.com/fabiocaccamo/django-admin-interface/commit/aec7ede184f03939b8476000d400f1050b367980).

**Pull request merged**